### PR TITLE
auth: fix rendering error and progress messages

### PR DESCRIPTION
**Pull request by saghul**
Also removed some no longer used styles.

### CODE REVIEW

**Comment by saghul** [comment]
Screen Shot 2019-11-29 at 12 47 16

Note the red error message. It wasn't rendered before.

File: react/features/authentication/components/styles.js [hunk 1]
```diff
@@ -26,25 +10,24 @@ export default createStyleSheet({
...
     /**
-     * The style of {@code LoginDialog}.
+     * The style used when an error message is rendered.
      */
-    loginDialog: {
-        ...dialog,
-        flex: 0,
-        flexDirection: 'column'
+    errorMessage: {
+        color: ColorPalette.red
```
Comment (zbettenbuk) [inline react/features/authentication/components/styles.js]: Don't you wanna brand this color?

Comment (saghul) [reply react/features/authentication/components/styles.js]: I'm not sure where to put it TBH.

Comment (zbettenbuk) [reply react/features/authentication/components/styles.js]: i'd put it in the top level definition with a name something like '`errorMessageText` or similar

**Comment by saghul** [comment]
@zbettenbuk Updated and made the error text themable. PTAL.

**Pull request merged**